### ISSUE TITLE
Missing accessibility labels

### ISSUE THREAD

**Issue by nkreer** [blokadaorg/blokada#692]
### Flawed behaviour

Blokada's UI is missing several accessibility labels for menu items, often resulting in confusion for visually impaired users who rely on screen reader or speech recognition tools to use their devices. Prominent examples for these issues are elements on the main view like the hamburger button or the _help_ and _share_ icons. This behaviour seems to be consistent throughout all locales. (Of course, added labels will have to be localised as well)

### Expected behaviour

Labels describing all elements should be present anywhere and considered a bug if missing. This was already dealt with in #98 almost two years ago but the labels are missing (again) at the moment.

### Steps to reproduce  

1. Enable TalkBack in Android's System Settings (Install from F-Droid if neccessary)
2. Launch Blokada and try to navigate

### Configuration
1. Blokada version: 4.5.2
2. Device model: Xiaomi beryllium
3. Android version / ROM: android-29 miui

I'd be happy to assist with further issues concerning Blokada accessibility.

**Comment by nkreer** [comment]
I would like to point out that, as this issue breaks the application for a user group of significant size, it should be considered a bug in the UI rather than a feature request. 

Thanks for your work on Blokada and making our lives better day after day 😄
